### PR TITLE
Move 5 nodes from 4.12 cluster pool into 4.14

### PR DESCRIPTION
**Pull request by lewisdenny**
This patch moves the following 5 nodes from the 4.12 cluster pool into
the 4.14 cluster pool.

- openstack-k8s-operators-cdc-vexx-01
- openstack-k8s-operators-cdc-vexx-02
- openstack-k8s-operators-cdc-vexx-03
- openstack-k8s-operators-cdc-vexx-04
- openstack-k8s-operators-cdc-vexx-05

JIRA: [OSPRH-2709](https://issues.redhat.com//browse/OSPRH-2709)

### CODE REVIEW

**Comment by lewisdenny** [comment]
/hold

Holding this until https://github.com/openshift/release/pull/47568 and https://github.com/openshift/release/pull/47567 are merged

File: clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml [hunk 1]
```diff
@@ -32,17 +42,17 @@ spec:
...
   platform:
     openstack:
       cloud: openstack
       credentialsSecretRef:
         name: openstack-operators-vexxhost-hive-creds
   pullSecretRef:
     name: pull-secret
-  runningCount: 9
-  size: 9
+  runningCount: 15
```
Comment (viroel) [inline clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml:54]: ok, 15 is fine

Comment (lewisdenny) [reply clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml:54]: Yeah I set to 15 as that's what 4.12 had before we started this migration :)

**Comment by viroel** [comment]
/hold cancel

we moved all projects to 4.14

**Pull request merged**